src/openforms/scss/vendor/_rjsf-field.scss

```diff
@@ -25,6 +25,11 @@
     input[type='email'] {
       width: $input-field-size; // override from admin/rjsf default
     }
+
+    select {
+      background-color: var(--form-input-bg) !important;
+      border-color: var(--border-color) !important;
+    }
```

Comment:
maybe try using `color-scheme: dark` here too and see how that reacts? If we can avoid the `!important`, we should.

Reply:
It gets a bit of a different colour:
![Screenshot from 2022-10-19 12-36-53](https://user-images.githubusercontent.com/19154114/196668541-c3db82f1-4228-46b1-b5d7-d8b6fc1a876a.png)
Should I still go for it or leave the the colours?

Reply:
Hrm, probably because we explicitly set the background for the text inputs rather than using `color-scheme: dark` for those. Let's go with the explicit colors then - I'm curious which styles overrides this color though that require the `!important`.